setup.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from setuptools import setup, find_packages
+
+with open('README.rst', 'r') as readme:
+    README_TEXT = readme.read()
+
+VERSION = '0.0.1dev'
+
+
+def write_version_py(filename=None):
+    if filename is None:
+        filename = os.path.join(
+            os.path.dirname(__file__), 'foam_wrapper', 'version.py')
+    ver = """\
+version = '%s'
+"""
+    fh = open(filename, 'wb')
+    try:
+        fh.write(ver % VERSION)
+    finally:
+        fh.close()
+
+
+write_version_py()
+
+setup(
+    name='foam_wrapper',
```

Comment:
We have started to develop a native data transfer wrapper while pythonFLU is missing some vital FOAM classes (pointMesh,...). We have already working unittest on that and we could put that package also to github. At this stage it is in the same branch as a separate package called "foam_tokenwrapper". I think that this naming  of the packages is one thing which we should have some guidelines and discuss that in the meeting.